backend_site/app.py

```diff
@@ -0,0 +1,52 @@
+"""
+The app module, containing the app factory function.
+Factory functions allow us to have multiple configurations of the server
+(testing, production, etc.)
+See http://flask.pocoo.org/docs/1.0/patterns/appfactories/ for more info
+"""
+
+from flask import Flask
+
+from backend_site.extensions import cors
+from backend_site.settings import ProdConfig
+
+from backend_site.events import blueprint as events_routes
+from backend_site.officers import blueprint as officers_routes
+from backend_site.projects import blueprint as projects_routes
+from .views import blueprint as static_views
+
+
+# def corsInit(routes, origins):
```

Comment:
Don't forget to remove old lines of code from making this PR if they aren't necessary